subprojects/gst-plugins-good/gst/rtp/gstrtph266pay.c

```diff
@@ -205,12 +266,36 @@ static void
 _set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec)
 {
+  GstRtpH266Pay *rtph266pay = GST_RTP_H266_PAY (object);
+
+  GST_OBJECT_LOCK (rtph266pay);
```

Comment:
Do you think it could lead to some kind of error or wrong behavior? I agree it isn't necessary, but if it's just because of that, I'd rather keep it as is. The code is simpler this way.